todo_app/test/test_viewmodel.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from todo_app.data.view_model import ViewModel
+
+class StubResponse():
```

Comment:
This class is not necessary.

The Item class is a very simple data class, there's no need to create Stub version of it. Import and use the real Item class.